x-pack/plugins/observability_solution/observability/common/custom_threshold_rule/get_view_in_app_url.ts

```diff
@@ -60,6 +62,7 @@ export const getViewInAppUrl = ({
     dataset,
     timeRange,
     query,
+    spaceId,
     filters: [...searchConfigurationFilters, ...groupFilters],
   });
```

Comment:
Instead `spaceId` should be a second arg like so: 

```
return logsExplorerLocator?.getRedirectUrl({
     dataset,
     timeRange,
     query,
     filters: [...searchConfigurationFilters, ...groupFilters],
   }, { spaceId } );
```

Because it isn't part of a specific locator state, but a param for building a URL